core/silkworm/common/endian.hpp

```diff
@@ -80,9 +67,9 @@ namespace le {
     }
 }  // namespace le
 namespace be {
-    static inline uint16_t load(uint16_t value) noexcept { return SILKWORM_BSWAP16(value); }
-    static inline uint32_t load(uint32_t value) noexcept { return SILKWORM_BSWAP32(value); }
-    static inline uint64_t load(uint64_t value) noexcept { return SILKWORM_BSWAP64(value); }
+    static inline uint16_t load(uint16_t value) noexcept { return intx::bswap(value); }
+    static inline uint32_t load(uint32_t value) noexcept { return intx::bswap(value); }
+    static inline uint64_t load(uint64_t value) noexcept { return intx::bswap(value); }
```

Comment:
Does intx already implement instrinsics ?

Reply:
Yes, it does: https://github.com/chfast/intx/blob/v0.7.1/include/intx/intx.hpp#L547